media-libs/ctl/files/ctl-1.5.2-Use-GNUInstallDirs-and-fix-PkgConfig-files-1.patch

```diff
@@ -243,7 +243,7 @@ index 1817071..eb7e14e 100644
  endif()
  
 -find_package( OpenEXR QUIET )
-+PKG_CHECK_MODULES ( OpenEXR REQUIRED OpenEXR )
++PKG_CHECK_MODULES ( OpenEXR QUIET OpenEXR )
```

Comment:
Did you mean to keep this changed?

Reply:
Yes I think so. The upstream way also uses QUIET and thus does not require OpenEXR to be present. The REQUIRED was introduced by dracwyrm in this patch.
Are there any arguments to switch it back?

Reply:
I would avoid changing patches used by the previous version unecessarily, but in this case I'm not sure it matters much.

Reply:
Do you mind if I drop it on merge?

Reply:
Nope, I haven't tested without the change, but it should work, as it has before.